### PR TITLE
started working on meetings

### PR DESCRIPTION
**Pull request by syh379**
- Added a POST route for meetings
- Added a GET route for getting meetings by their ID

### CODE REVIEW

File: src/app/api/meeting/[id]/route.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { NextResponse, NextRequest } from "next/server";
+import { PrismaClient, Meeting } from "@prisma/client";
+const prisma = new PrismaClient();
+
+export async function GET(
+  request: NextRequest,
+  { params }: { params: { id: string } },
+): Promise<NextResponse<Meeting | null>> {
+  // If the identifier starts with meeting_, the it is an ID, otherwise it is a slug
```
Comment (aidansunbury) [inline src/app/api/meeting/[id]/route.ts:9]: Delete this comment

File: src/app/api/meeting/[id]/route.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  });
+
+  // If the meeting does not exist, return a 400 error
+  if (!meeting) {
+    return NextResponse.json(meeting, {
+      status: 400,
+    });
+  }
+
+  // If the user exists, return the user with a 200 status code
```
Comment (aidansunbury) [inline src/app/api/meeting/[id]/route.ts:27]: Make the comment say "if the meeting exists"

**Review by aidansunbury — CHANGES_REQUESTED** [review]
Just a few comments to update

File: src/app/api/meeting/[id]/route.ts (hunk 1 above)
Comment (syh379) [reply src/app/api/meeting/[id]/route.ts]: updated

File: src/app/api/meeting/[id]/route.ts (hunk 2 above)
Comment (syh379) [reply src/app/api/meeting/[id]/route.ts]: updated

**Review by aidansunbury — APPROVED** [review]
lgtm!

**Pull request merged**